From: Sven Eckelmann <sven@narfation.org>
Date: Thu, 13 Aug 2020 16:20:10 +0200
Subject: smatch: disable verbose check_unused_ret

This check creates so many false positive that it doesn't make a lot of
sense of having it around.

Signed-off-by: Sven Eckelmann <sven@narfation.org>

diff --git a/check_unused_ret.c b/check_unused_ret.c
index a7caa823f6d72861a54c99216450d459098a9b2e..0e133ac2be60683f7fd3f63c3b648afce280d94f 100644
--- a/check_unused_ret.c
+++ b/check_unused_ret.c
@@ -203,6 +203,9 @@ void check_unused_ret(int id)
 {
 	my_id = id;
 
+	/* too many bogus errors */
+	return;
+
 	/* It turns out that this test is worthless unless you use --two-passes.  */
 	if (!option_two_passes)
 		return;
